packages/devextreme/js/ui/gallery.js

```diff
@@ -146,7 +146,10 @@ const Gallery = CollectionWidget.inherit({
             */
 
 
-            _itemAttributes: { role: 'option' },
+            _itemAttributes: {
+                role: 'option',
+                'aria-label': 'gallery item'
```

Comment:
[base approach]
We should use localization variables for labels

Reply:
fixed